src/extension.ts

```diff
@@ -78,10 +79,18 @@ function getPRsPerSHAs() {
   watermelonPanel.currentPanel?.doRefactor({
     command: "loading",
   });
+
+  // Splice the array to avoid the 256 character restriction on the query to the GitHub API
+  if (arrayOfSHAs.length > 22) {
+    arrayOfSHAs.splice(22, arrayOfSHAs.length - 22);
```

Comment:
This modifies the array, could we use a copying method such as [slice](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice)

Reply:
Slice also allows slicing above the number of elements on the array without breaking, making the if innecesary